test/e2e/test_helm_install_upgrade.go

```diff
@@ -62,85 +62,16 @@ func TestHelmUpgrade(t *testing.T) {
 		assertionsAfterInstall []assertion
 		assertionsAfterUpgrade []assertion
 	}{
-		// NOTE: We do not support versions earlier than 1.2 with the helm chart.
-		// The initial version of the chart contained CRDs from KGO 1.2. which
-		// introduced a breaking change which makes it impossible to upgrade from
-		// automatically (without manually deleting the CRDs).
```

Comment:
I guess this could have been left as is but meh. We can also remove it.

Reply:
I think that we could document this in the chart's `UPGRADE.md` as that's where users would seek such information, WDYT?